fe/src/pages/Home/HomeContent.tsx

```diff
@@ -129,23 +132,31 @@ const PRODUCTS_CARD_ITEMS: Array<ProductCardItem> = [
 ];
 
 function HomeContent(): JSX.Element {
-  const { view } = React.useContext(HomeContext);
+  const { view, page } = React.useContext(HomeContext);
   const isViewSwitchable: boolean = useBreakpointValue({ base: false, lg: true }) || false;
 
+  const pageSize = 21;
```

Comment:
this is a small thing, but maybe make this number a variable like this:
const ITEM_COUNT_ON_PAGE = 21;
and answer would be this const pageSize = ITEM_COUNT_ON_PAGE;

Reply:
> this is a small thing, but maybe make this number a variable like this:
> `const ITEM_COUNT_ON_PAGE = 21;`
> and answer would be this
> `const pageSize = ITEM_COUNT_ON_PAGE;`

I think it will make more sense to just rename the variable to `ITEM_COUNT_ON_PAGE` then. 🤔

Reply:
@tomimarkus991 Updated the variable for now, i think it's not super useful to keep it separate for now, at least until we add custom page sizing.

I didn't add paginator for the bottom, didn't look right when there were few products left on the last page. For now i think its fine, i don't think its a big hassle to scroll up to swap page for now.

But i did swap Paginator with ViewToggler, see what you think about the new styling in Alberts comment reply ^.